consensus/consensus-types/src/block_test_utils.rs

```diff
@@ -182,9 +180,7 @@ pub fn block_forest_and_its_keys(
 // Using current_timestamp in this test
 // because it's a bit hard to generate incremental timestamps in proptests
 pub fn get_current_timestamp() -> Duration {
```

Comment:
same here, replace calls to get_current_timestamp by time::duration_since_epoch()

Reply:
Ahahah, I hesitated and decided that `get_current_timestamp` was a clearer name and let it. But I can remove it, sure.